weather-app/src/app/footer/footer.component.scss

```diff
@@ -1,8 +1,10 @@
+@import 'src/assets/styles/_general';
+
 .footer {
   display: flex;
   align-items: center;
   justify-content: center;
-  background-color: #eaefea;
+  background-color: $base-color;
```

Comment:
Why did you correcte this part? I think, It could be in next bratch or you can delegate it to Alex.